app/views/middleware_topology/show.html.haml

```diff
@@ -10,7 +10,7 @@
         %svg.kube-topology
           %g.MiddlewareServer{:transform => "translate(21, 21)"}
             %circle{:r => "17"}
-            %image{:height => "20", :width => "20", :x => "-9", "xlink:href" => "/assets/100/middleware_server.png", :y => "-9"}
+            %image{:height => "20", :width => "20", :x => "-9", "xlink:href" => "/assets/svg/vendor-wildfly.svg", :y => "-9"}
```

Comment:
Certainly not totally broken, as it works here :-)  At the moment the `vendor-wildfly` icon is the fallback for a generic MW-server. And those icons are for the show/hide icons, so there is no need to dynamically determine them.